src/Platforms/SqlsrvSqlPlatform.php

```diff
@@ -11,7 +11,7 @@
 use Nextras\Dbal\Connection;
 
 
-class MssqlSqlPlatform implements IPlatform
+class SqlsrvSqlPlatform implements IPlatform
```

Comment:
Sorry for the confusion, but this sould stay as it was (with little typo). It not dependes on the driver, it dependes on the actual database server, so **MssqlPlatform** should be the right name.